task3/src/data-access/index.js

```diff
@@ -0,0 +1,25 @@
+import Sequelize from 'sequelize';
```

Comment:
Do we really need data-access folder just to have sequelize initialized? Or should there be something else?

Reply:
Mostly depends on individual project structures, no common pattern. Sometimes DB is initialized in some folder called initializers or setup because usually there are plenty of set up files in a project. Sometimes it's data-access.

Reply:
Nice, I'll try something alike with dotenv.config()